data/sql/nulls/nulls_test.go

```diff
@@ -45,6 +46,9 @@ func TestScanFail(t *testing.T) {
 	for _, test := range tests {
 		inst := test.GetInst()
 		err := inst.(sql.Scanner).Scan(test.FailParam)
+		if err == nil {
```

Comment:
this doesnt seem right. its bypassing the actual tests.

Reply:
in this scenario we pass the fail param which should cause the process to fail with an expected error, if err is nil then the tests should fail.